index.html

```diff
@@ -46,5 +47,178 @@ <h1>I’m Rashmi.<br> to see<br> you!</h1>
                 </ul>
             </div>
         </div>
+
+        <section class="work">
+            <div class="grid-container">
+                <div class="grid-item">
+                    <img src="img/Snapshoot Portfolio.png">
+                    <div class="detail">
+                        <h1>Tonic</h1>
+                        <div class="list-01">
+                            <ul>
+                                <li class="canopy">CANOPY</li>
+                                <li><img src="img/Counter.png"></li>
+                                <li>Back End Dev</li>
+                                <li><img src="img/Counter.png"></li>
+                                <li>2015</li>
+                            </ul>
+                        </div>
+                        <p>A daily selection of privately<br> 
+                            personalized reads; no accounts or<br> 
+                            sign-ups required.
+                        </p>
+                        <div class="lang">
+                            <ul>
+                                <li>HTML</li>
+                                <li>CSS</li>
+                                <li>JavaScript</li>
+                            </ul>
+                        </div>
+                        <div >
+                             <button class="button-A">See project</button>
+                        </div>
+                    </div>
+      
```

Comment:
Kindly, I suggest you remove all unnecessary blank lines in your code base to make your project look more professional and for best practice reasons. 😄